sys-boot/systemrescuecd-x86-grub/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer>
+		<name>Michael Lange</name>
+		<email>gentoobugs@milaw.biz</email>
```

Comment:
Well, I suggest you add something like:

    <description>Accepts only Bugzilla mail, do not e-mail directly</description>

to avoid unnecessary frustration ;-).